edge/evtstreams/cpu2evtstreams/README.md

```diff
@@ -365,10 +360,6 @@ sudo docker ps
 14. On any machine, subscribe to the Event Streams topic to see the json data that cpu2evtstreams is sending:
   - If using IBM Event Streams in IBM Cloud:
```

Comment:
Did you want to also delete this line?

Reply:
Oh shoot, totally thought I got all of those. I'll remove it and commit again